package.json

```diff
@@ -129,7 +129,11 @@
     "truffle": "^4.1.11",
     "truffle-contract": "^3.0.5",
     "uglify-es": "github:mishoo/UglifyJS2#harmony",
-    "web3": "^0.20.1"
+    "uport-connect": "^0.6.0",
```

Comment:
Alexander, could you please have a look into PR https://github.com/ChronoBank/ChronoMint/pull/691/files ?

Now we have multiple package.json files:
<root>/package.json
<root>/packages/core/package.json
<root>/packages/login/package.json
<root>/packages/login-ui/package.json

Added dependencies int his PR are moved into @chronobank/core and @chronobank/login pacakges.
Could you revert your changes in this file?